packages/web-components/src/components/masthead/top-nav.ts

```diff
@@ -158,65 +152,54 @@ class DDSTopNav extends StableSelectorMixin(HostListenerMixin(BXHeaderNav)) {
   /**
    * Handles `click` event on the left-hand paginator button.
    */
-  private _pagenateLeft() {
+  private _paginateLeft() {
     const {
-      _caretLeftNode: caretLeftNode,
-      _caretRightNode: caretRightNode,
       _contentContainerNode: contentContainerNode,
       _currentScrollPosition: currentScrollPosition,
-      _isIntersectionRightTrackerInContent: isIntersectionRightTrackerInContent,
       _slotNode: slotNode,
     } = this;
     const elems = slotNode?.assignedElements() as HTMLElement[];
     if (elems) {
-      const caretLeftNodeWidthAdjustment = caretLeftNode?.offsetWidth ?? 0;
-      const caretRightNodeWidthAdjustment = isIntersectionRightTrackerInContent ? caretRightNode!.offsetWidth : 0;
-      const currentFirstVisibleElementIndex = elems.findIndex(elem => elem.offsetLeft >= currentScrollPosition);
+      const currentFirstVisibleElementIndex = elems.findIndex(elem => elem.getBoundingClientRect().left + elem.offsetWidth >= 80);
```

Comment:
Where does the magic number `80` come from? My question comes especially because `elem.getBoundingClientRect().left` is the left position from the viewport's left position.

Reply:
oh oops accidentally left that in. it should be `caretRightNodeWidthAdjustment  + caretLeftNodeWidthAdjustment` with adding the variables as follow: 
```
      const caretRightNodeWidthAdjustment = caretRightNode?.offsetWidth ?? 0;
      const caretLeftNodeWidthAdjustment = !isIntersectionLeftTrackerInContent ? caretLeftNode!.offsetWidth : 0;
      const currentFirstVisibleElementIndex = elems.findIndex(elem => elem.getBoundingClientRect().left + elem.offsetWidth >= caretRightNodeWidthAdjustment  +caretLeftNodeWidthAdjustment);
```

Reply:
I see thanks @ariellalgilmore - Probably did you meant:

```javascript
elem.getBoundingClientRect().right >= caretRightNodeWidthAdjustment  + caretLeftNodeWidthAdjustment
```

Reply:
yeah thanks!